PProPanel/ext.js

```diff
@@ -222,11 +222,11 @@ function loadJSX() {
 	var extensionPath = csInterface.getSystemPath(SystemPath.EXTENSION);
 
 	// load general JSX script independent of appName
-	const extensionRootGeneral = extensionPath + '/jsx/';
+	var extensionRootGeneral = extensionPath + '/jsx/';
```

Comment:
>For my panel projects, I've set up a boilerplate using create-react-app, because reactJS is the ui framework we use at work.

Neat. We use [Vue](https://vuejs.org/).

>Hmm... the fact that ES specs differ between versions of CEP really just drives me to hold on to the extra compile step using babel, as this will maximize compatibility and it allows me to use the new cool things of JavaScript like arrow functions, spread operator and so on ;-)

I hear you there, but you're not committing to an environment that runs with babel. The environment you're committing to _does_ support `const` (especially as used within this context - i.e. non-strict mode), so it would be preferable to make use of it. :) (Pending, of course, direction from @bbb999...)